Rock, paper, scissors/script.js

```diff
@@ -30,26 +30,10 @@ function getPlayerChoice() {
     }
   }
 
-let playerSelection = getPlayerChoice()
-let computerSelection = getComputerChoice().toLowerCase();
-
-/*
-Now it is time to write a function that will cover the entire round of the game
-*/
-
-function playRound(playerSelection, computerSelection) {
-    if (playerSelection === computerSelection) {
-        return `Computer chose ${playerSelection} too. It's a tie!`;
-    } else if (playerSelection === "rock" && computerSelection === "scissors" || 
-               playerSelection === "paper" && computerSelection === "rock" ||
-               playerSelection === "scissors" && computerSelection === "paper") {
-        return `Computer chose ${computerSelection} \nYou win! ${playerSelection} beats ${computerSelection}`;
-    } else { 
-        return `Computer chose ${computerSelection} \nYou lose! ${computerSelection} beats ${playerSelection}`;
-    }
-  }
- 
-//Finally, we find the winner and print the result of completed round into DevTools console
+function getComputerSelection() {
+    const choices = ["rock", "paper", "scissors"];
+    const randomIndex = Math.floor(Math.random() * choices.length);
+    return choices[randomIndex];
+}
 
-const roundResult = playRound(playerSelection, computerSelection);
-console.log(roundResult);
\ No newline at end of file
+main()
```

Comment:
Обрати внимание, что последняя строка у тебя не заканчивается символом переноса строки.
На некоторых системах и в некоторых языках это может быть проблемой, поэтому традиционно файл заканчивают переносом строки (чтобы все строки заканчивались этим символом).
```suggestion
main()

```